proto/options.proto

```diff
@@ -137,14 +137,14 @@ message Options {
   enum Action {
     route = 1;
     locate = 2;
-    sources_to_targets = 3;
-    optimized_route = 4;
-    isochrone = 5;
-    trace_route = 6;
-    trace_attributes = 7;
-    height = 8;
-    transit_available = 9;
-    expansion = 10;
+    sources_to_targets = 6;
```

Comment:
this seems unrelated to the pr

Reply:
looks like you reverted at least part of #2545

Reply:
actualy i guess it looks like you reverted all of it :smile:

Reply:
thanks for catching that